package.json

```diff
@@ -22,20 +27,26 @@
   },
   "homepage": "https://github.com/balena-io-modules/docker-toolbelt#readme",
   "dependencies": {
-    "@types/bluebird": "^3.5.30",
-    "@types/dockerode": "^2.5.24",
+    "@types/chai": "^4.2.22",
+    "@types/dockerode": "^3.3.0",
+    "@types/event-stream": "^3.3.34",
+    "@types/randomstring": "^1.1.8",
+    "@types/sinon": "^10.0.6",
+    "@types/tar-stream": "^2.1.0",
```

Comment:
All `@types` packages should be in the `devDependencies`, no? They're not needed in a production install, even if the respective packages are in the `dependencies` section.

Reply:
@pdcastro Good catch, the types used for testing should be under `devDependencies`, but some of the other types are required as dependencies so that the current types can be used by projects using this package (e.g. dockerode)